rules/salary-sacrifice-electronic-devices/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 seoDescription: Pay less tax on electronic devices by salary sacrificing with your pre-tax money and save approximately 30% depending on your tax bracket.
 type: rule
-title: Do you take advantage of salary sacrificing electronic devices?
+title: Paryoll - Do you take advantage of salary sacrificing electronic devices? (Aussie Only)
```

Comment:
Hi @JimmyChenSSW 

On all "(Aussie only)" the word "only" should be in lowercase

- [ ] 1. Please fix all

Reply:
Done - thank you!